src/cli/commands/link.js

```diff
@@ -58,7 +58,8 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
 
     const linkLoc = path.join(config.linkFolder, name);
     if (await fs.exists(linkLoc)) {
-      reporter.warn(reporter.lang('linkCollision', name));
+      const linkTarget = await fs.readlink(linkLoc);
+      reporter.warn(reporter.lang('linkCollision', name, linkTarget));
```

Comment:
I think we can improve the message even further by using `path.resolve` to have an absolute path rather than a relative path, what do you think?